users/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db       import models
+
+from core.models     import TimeStampModel
+from products.models import Product
+from stocks.models   import Stock
+
+class User(TimeStampModel): 
+    name     = models.CharField(max_length=300)
+    email    = models.CharField(max_length=300, unique = True)
+    password = models.BinaryField(max_length=300)
+    phone    = models.CharField(max_length=100, null = True)
```

Comment:
`=` 앞뒤 간격은 전체적으로 통일해주세요.
```suggestion
    phone    = models.CharField(max_length=100, null=True)
```

Reply:
수정 했습니다.